modules/engage-ui/src/main/resources/ui/js/app/engage-ui.js

```diff
@@ -541,6 +541,12 @@ function($, bootbox, _, alertify, jsyaml) {
     });
 
     $($oc_sort_dropdown).on('change', function() {
+      let epFrom = GetURLParameter('epFrom');
+      if (epFrom) {
+        $('#oc-search-form .form-group').append(
+          '<input type=\'hidden\' name=\'epFrom\' value=\'' + epFrom + '\' />'
```

Comment:
This allows adding unescaped user input. This means that I could generate malicious links and send them to users, which then have my code executed if they click on it. Iirc, there should already be an escape method somewhere.

Reply:
🤦 OMG! Sorry, that should not happen. Lesson learned. After a hard day of meetings, you shouldn't fix a bug in a hurry.
Is there a way to restart the CI pipeline? The build failed on a module that has no modifications.

Reply:
Ignore that issue for now. Merging `r/9.x` into `r/10.x` just broke  something. Will be fixed as soon as the tests on #2922  succeed and that one gets merged.